CHANGELOG.rst

```diff
@@ -0,0 +1,34 @@
+Changelog
+========================================================================
+
+in development
```

Comment:
I think there is nothing under "in development" at this moment.

And the changes below correspond to the `v0.8.0` per current `package.json`
https://github.com/StackStorm/st2chatops/blob/5b5fe490ee4ff510d56b4b3147133c33748d697d/package.json#L3